server/schedule/operator_controller.go

```diff
@@ -899,5 +899,6 @@ func (oc *OperatorController) GetLeaderScheduleStrategy() core.ScheduleStrategy
 func (oc *OperatorController) RemoveStoreLimit(storeID uint64) {
 	oc.Lock()
 	defer oc.Unlock()
+	oc.cluster.AttachAvailableFunc(storeID, nil)
```

Comment:
why is `nil` rather than `func(){ return false }`

Reply:
I just reset to the original value when the store limit does not exist.